doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
```

Comment:
I think there's still an important distinction to make here. I think an Inventory Plan is a set of rules that _initially_ defines which Theme a Content Item belongs to. The Content Item should be able to move between themes independent of the rules defining the Inventory Plan.

The distinction is important, because we **should not** expect to find all Content Items belonging to a Theme by applying the rules set out in the Inventory Plan.

Reply:
Content Items do not move across themes, only the rules on the inventory plan do.